openlibrary/core/processors/readableurls.py

```diff
@@ -161,9 +165,13 @@ def match(path):
     if thing:
         prefix = thing.key
 
-    if thing and thing.type.key == type:
-        title = thing.get(property) or default_title
-        middle = '/' + h.urlsafe(title.strip())
+    if thing and thing.type.key == _type:
+        title = thing.get(_property) or default_title
+        try:
+            from urllib.parse import quote_plus
```

Comment:
```suggestion
            from six.moves.urllib_parse import quote_plus
```
And please move the import to the top of the file.